risc0/zkp/src/verify/mod.rs

```diff
@@ -424,12 +445,13 @@ where
         return Err(VerificationError::InvalidProof);
     }
 
-    // Set the mix mix value
+    // Set the mix mix value, pseudorandom value used for FRI batching
     let mix = H::ExtElem::random(&mut iop);
     // debug!("mix = {mix:?}");
 
     // Make the mixed U polynomials.  combo_u has one element for each
     // back for each combo, plus 1 for the check.
+    // TODO what is combo_u?
```

Comment:
So if you look at the DEEP-ALI equation, you can see that the denominator is shared for all columns which have the same set of taps.  If you group all terms with the same denominator, you can reduce the number of inverses you need to compute, so basically a 'combo' is a group of columns that share the same structure.